go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/golangci/golangci-lint
 
-go 1.16
+go 1.17
```

Comment:
1.18?

Reply:
not for the first version, we need to stay compatible as much as possible.